embedded-tests/src/test/java/org/apache/druid/testing/embedded/k8s/K3sClusterResource.java

```diff
@@ -74,18 +74,18 @@ public class K3sClusterResource extends TestcontainerResource<K3sContainer>
   public static final String DRUID_NAMESPACE = "druid";
   private static final String NAMESPACE_MANIFEST = "manifests/druid-namespace.yaml";
 
-  private static final String COMMON_CONFIG_MAP = "druid-common-props";
-  private static final String SERVICE_CONFIG_MAP = "druid-%s-props";
+  protected static final String COMMON_CONFIG_MAP = "druid-common-props";
+  protected static final String SERVICE_CONFIG_MAP = "druid-%s-props";
 
   public static final long POD_READY_TIMEOUT_SECONDS = 300;
 
-  private final List<File> manifestFiles = new ArrayList<>();
-  private final List<K3sDruidService> services = new ArrayList<>();
+  protected final List<File> manifestFiles = new ArrayList<>();
+  protected final List<K3sDruidService> services = new ArrayList<>();
 
-  private KubernetesClient client;
-  private String druidImageName;
+  protected KubernetesClient client;
+  protected String druidImageName;
 
-  private final Closer closer = Closer.create();
+  protected final Closer closer = Closer.create();
```

Comment:
We should keep these fields private.
We can expose methods if needed.

Reply:
Fixed.